R/metab.bookkeep.R

```diff
@@ -113,7 +113,7 @@ metab.bookkeep <- function(do.obs, do.sat, k.gas, z.mix, irr, ...){
 
 	R <- mean(nep.night, na.rm=TRUE) * freq # should be negative
 	NEP <- mean(delta.do.metab, na.rm=TRUE) * freq # can be positive or negative
-	GPP <- mean(nep.day, na.rm=TRUE) * sum(dayI) - R # should be positive
+	GPP <- mean(nep.day, na.rm = TRUE) * sum(dayI) - mean(nep.night, na.rm = TRUE) * sum(dayI) # should be positive
```

Comment:
Curious, would this be same as `GPP <- NEP - R`? They wouldn't necessarily be equal, but the difference should just be whether you think there is any GPP that might happen outside the designated daytime hours. I prefer this formulation for some philosophical reasons, but I think the distinction is a red herring and I don't really oppose the "GPP only happens during dayI" perspective.

Also, isn't `mean(nep.day) * sum(dayI) == sum(nep.day)` ? If so, the latter (`sum(nep.day)`) is syntactically more concise and computationally more efficient.